homeassistant/components/mastodon/config_flow.py

```diff
@@ -46,7 +47,7 @@
 class MastodonConfigFlow(ConfigFlow, domain=DOMAIN):
     """Handle a config flow."""
 
-    VERSION = 1
+    VERSION = 2
```

Comment:
Let's use minor_version. This way people can still rollback to 2024.8 if something else doesn't work like expected